src/utilities/install.ts

```diff
@@ -8,11 +8,13 @@ import ora from 'ora';
 
 /**
  * It installs dependencies from a package.json file
- * @param {'skip' | 'npm' | 'yarn'} choice - The package manager to use.
- * @param {string} name - The name of the project
- * @returns a promise.
+ * @param choice - The package manager to use.
+ * @param name - The name of the project
  */
-export async function installDeps(choice, name) {
+export async function installDeps(
+	choice: 'skip' | 'npm' | 'yarn',
```

Comment:
could we lift this into a new type called PackageManagerChoice = 'skip' | 'npm' | 'yarn',
this type is stated twice here and https://github.com/sern-handler/cli/pull/74/files#r963771486